server/pages/appointments/attendanceOutcomePage.ts

```diff
@@ -61,7 +61,7 @@ export default class AttendanceOutcomePage extends BaseAppointmentUpdatePage {
     return contactOutcomes.map(outcome => ({
       text: outcome.name,
       value: outcome.id,
```

Comment:
I think this will need to be code as well

Reply:
We'll also need to update this line to code: https://github.com/ministryofjustice/hmpps-community-payback-ui/blob/ee197e00a2af8b5e2623160d325ca403faa3972e/server/pages/appointments/attendanceOutcomePage.ts#L23

I might do a double check in the app once that's changed, just incase we've missed anything in tests